packages/components/pie-cookie-banner/src/localisation-utils.ts

```diff
@@ -38,16 +38,17 @@ function getObjectKeyValue (obj:object, path:string):string {
     const travel = (regexp:RegExp) => String.prototype.split
         .call(path, regexp)
         .filter(Boolean)
-        .reduce((acc:Record<string, unknown>, key:string) => {
-            if (acc !== null && acc !== undefined) {
+        // eslint-disable-next-line @typescript-eslint/no-explicit-any
+        .reduce((acc:{[key: string]: any}, key:string) => {
```

Comment:
could it be worth asking some of the TS engineers over at skip for their thoughts on using `any` here?

Reply:
I agree, but it would be good to get this merged to unblock the build for now